src/app/auth/store/auth.effects.ts

```diff
@@ -0,0 +1,27 @@
+import {Injectable} from "@angular/core";
+import {Actions, concatLatestFrom, createEffect, ofType} from "@ngrx/effects";
+import {catchError, map, mergeMap, of, tap} from "rxjs";
+import {AuthService} from "../services/auth.service";
+import {requestLogin, requestLoginFail, requestLoginSuccess} from "./auth.actions";
```

Comment:
```suggestion
import * as AuthActions from "./auth.actions";
```
Quite often such way of import is used for actions.